starfish/experiment/builder/__init__.py

```diff
@@ -157,7 +157,7 @@ def write_experiment_json(
 
     experiment_doc: Dict[str, Any] = {
         'version': str(CURRENT_VERSION),
-        'auxiliary_images': {},
```

Comment:
pardon my ignorance, but what does shoving auxiliary_images into images buy you?